examples/multi-module/pom.xml

```diff
@@ -0,0 +1,56 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Defines artifact information for this parent POM. -->
+    <groupId>com.example</groupId>
+    <artifactId>jib-multimodule</artifactId>
+    <packaging>pom</packaging>
+    <version>0.1.0</version>
+    <name>Jib Multi-module Example</name>
+
+    <!-- Inherits from the Spring Boot Starter POM. -->
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.3.RELEASE</version>
+    </parent>
+
+    <!-- The modules that are part of this project. -->
+    <modules>
+        <module>name-service</module>
+        <module>hello-service</module>
+    </modules>
+
+    <!-- Uses Java 8. -->
+    <properties>
+        <java.version>1.8</java.version>
+
+        <!-- IMPORTANT: Set the environment variable PROJECT_ID to your own Google Cloud Platform project. -->
+        <image>gcr.io/${env.PROJECT_ID}/${artifactId}:${version}</image>
```

Comment:
```
[WARNING] The expression ${artifactId} is deprecated. Please use ${project.artifactId} instead.
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```